mkgu_packaging/dicarlo/marques/marques_ringach2002.py

```diff
@@ -0,0 +1,88 @@
+
+import numpy as np
+from brainio_collection.packaging import package_data_assembly
+import scipy.io as sio
+from brainio_base.assemblies import DataAssembly
+
+DATA_DIR = '/braintree/data2/active/users/tmarques/bs_datasets/Ringach2002.mat'
+ASSEMBLY_NAME = 'dicarlo.Marques2020_Ringach2002'
+ORIENTATION_STIM_NAME = 'dicarlo.Marques2020_orientation'
+PROPERTY_NAMES = ['baseline', 'maxdc', 'mindc', 'maxac', 'mod_ratio', 'circ_var', 'bandwidth', 'orth_pref_ratio',
+                  'or_sel', 'cv_bw_ratio', 'opr_cv_diff', 'opr_bw_ratio']
```

Comment:
as above, please write these out to avoid confusion

Reply:
One of them is going to be 'orthgonal_preferred_ratio_bandwidth_ratio'

Reply:
that's okay, it only takes up space in code, right? 
(For a benchmark identifier to show on the website I'd agree to shorten it)